core/src/main/java/org/elasticsearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -169,8 +168,8 @@ public static DisMaxQueryBuilder fromXContent(QueryParseContext parseContext) th
             }
         }
 
-        if (!queriesFound) {
-            throw new ParsingException(parser.getTokenLocation(), "[dis_max] requires 'queries' field");
+        if (!queriesFound || queries.isEmpty()) {
```

Comment:
doesn't this beak bw comp as opposed to other changes in this PR? I thought we'd have to bubble the empty up in this case as well.